popsycle/synthetic.py

```diff
@@ -676,9 +759,13 @@ def _make_comp_dict(iso_dir, log_age, currentClusterMass, star_dict, next_id,
     NSKickSpeed : float or int
         Kick speed of NS (in km/s)
 
-    set_random_seed : bool
-        Forces PyPopStar to fix the random seed to 42,
-        enforcing identical output.
+    debug : bool
+        If set to True, removes all random sampling and forces identical
```

Comment:
I thought we were removing debug and using random_seed instead?

Reply:
See comment below